src/pods/canvas/components/table/database-table-border.component.tsx

```diff
@@ -0,0 +1,29 @@
+import { TABLE_CONST } from '@/core/providers';
+import { TABLE_BORDER_RADIUS } from './database-table.const';
+import classes from './database-table.module.css';
```

Comment:
We could break down CSS in several files inlcluding only the specific part for each component.

If you find this interesting we can create an issue with some guidance and assign it to a student.